internal/vm/setup.go

```diff
@@ -74,6 +73,114 @@ _G["load_plugin_schema"] = function(plugin_schema_string)
     return nil, "error initializing schema for plugin: " .. err
   end
 end
+
+-- Remove functions from a schema definition so that
```

Comment:
In order to remove the need for extra patching these functions were copied from `kong/api/api_helpers.lua` and `kong/api/routes/kong.lua` which included multiple required modules and contained c bindings.

Reply:
Upstream Kong is tolerant of patches. We could send a patch to move these functions into a different lua file, which this code directly consumes. Only possible if there are no changes  in these functions (that can't be refactored upstream).